index.md

```diff
@@ -40,7 +40,7 @@ This project wouldn't exist without the support of [our incredible sponsors](htt
 <br />
 
 <a href="https://www.openiddictcomponents.com/">
-  <img src="https://www.openiddictcomponents.com/media/kz1aymji/openiddict-components-logo-poweredbyrsk.png" width="400px" alt="OpenIddict Components Logo" />
+  <img src="https://www.identityserver.com/media/xl4bf02t/openiddict-components-logo-poweredbyrsk-png.png" width="400px" alt="OpenIddict Components Logo" />
```

Comment:
Thanks for your PR!

I don't mind taking this change as-is, but isn't that a bit weird to have an "OpenIddict components" logo hosted by the same website that is dedicated to your Duende IdentityServer-based offerings? It might be confusing to some users.